client/me/help/main.jsx

```diff
@@ -73,32 +80,61 @@ const Help = React.createClass( {
 				<CompactCard className="help__support-link" href="https://support.wordpress.com/" target="__blank">
 					<div className="help__support-link-section">
 						<h2 className="help__support-link-title">{ this.translate( 'All support articles' ) }</h2>
-						<p className="help__support-link-content">{ this.translate( 'Looking to learn more about a feature? Our docs have all the details.' ) }</p>
+						<p className="help__support-link-content">
+							{ this.translate( 'Looking to learn more about a feature? Our docs have all the details.' ) }
+						</p>
 					</div>
 				</CompactCard>
 				<CompactCard className="help__support-link" href="https://dailypost.wordpress.com/" target="__blank">
 					<div className="help__support-link-section">
 						<h2 className="help__support-link-title">{ this.translate( 'The Daily Post' ) }</h2>
-						<p className="help__support-link-content">{ this.translate( 'Get daily tips for your blog and connect with others to share your journey.' ) }</p>
+						<p className="help__support-link-content">
+							{ this.translate( 'Get daily tips for your blog and connect with others to share your journey.' ) }
+						</p>
 					</div>
 				</CompactCard>
 				<CompactCard className="help__support-link help__support-link-contact" href="/help/contact/">
 					<div className="help__support-link-section">
 						<h2 className="help__support-link-title">{ this.translate( 'Get in touch' ) }</h2>
-						<p className="help__support-link-content">{ this.translate( 'Can\'t find the answer? Drop us a line and we\'ll lend a hand.' ) }</p>
+						<p className="help__support-link-content">
+							{ this.translate( 'Can\'t find the answer? Drop us a line and we\'ll lend a hand.' ) }
+						</p>
 					</div>
 					<Button className="help__support-link-button" primary>{ this.translate( 'Contact Us' ) }</Button>
 				</CompactCard>
 		</div>
 		);
 	},
 
+	getCoursesTeaser: function() {
+		return (
+			<div className="help__course-teaser">
+				<Card href="#/help/courses">
+					<Gridicon className="help__course-teaser-icon" icon="help" size={ 30 } />
```

Comment:
I changed it to `36` in @6c9d969

@mrheston Do you have any issues with the look of the button now?